From 8568e0da70806f211d6e2616e2a0c5ad8077efce Mon Sep 17 00:00:00 2001
From: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Date: Tue, 22 Jul 2025 10:39:36 +0800
Subject: [PATCH 02/32] arm: ast2700: Update WDT reset event log bit

Below should be the corret WDT reset evnet log bit in SCU1_080.

WDT full chip reset: SCU1_080[0]
WDT CA35 ARM reset : SCU1_080[1]
WDT SoC reset      : SCU1_080[2]
WDT SW reset       : SCU1_080[3]

Signed-off-by: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Change-Id: I67f5ea40dce98d4639508acb10b609025f193e99
(cherry picked from commit eb89c1b8ae557bb720b92002eea5e4cfecf88cc1)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 arch/arm/mach-aspeed/ast2700/cpu-info.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-aspeed/ast2700/cpu-info.c b/arch/arm/mach-aspeed/ast2700/cpu-info.c
index 61853a0f35..81351ae7a4 100644
--- a/arch/arm/mach-aspeed/ast2700/cpu-info.c
+++ b/arch/arm/mach-aspeed/ast2700/cpu-info.c
@@ -50,9 +50,9 @@ void ast2700_print_soc_id(void)
 #define SYS_SRST		BIT(0)
 
 #define WDT_RST_BIT_MASK(s)	(GENMASK(3, 0) << (s))
+#define BIT_WDT_FULL(s)		(BIT(0) << (s))
+#define BIT_WDT_ARM(s)		(BIT(1) << (s))
 #define BIT_WDT_SOC(s)		(BIT(2) << (s))
-#define BIT_WDT_FULL(s)		(BIT(1) << (s))
-#define BIT_WDT_ARM(s)		(BIT(0) << (s))
 #define BIT_WDT_SW(s)		(BIT(3) << (s))
 
 void ast2700_print_wdtrst_info(void)
-- 
2.34.1

